Index: app/src/main/java/com/example/projecte12/RetroFit.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.projecte12\r\n\r\nimport retrofit2.Retrofit\r\nimport retrofit2.converter.gson.GsonConverterFactory\r\n\r\nobject RetroFit {\r\n    private const val BASE_URL = \"http://10.0.2.2:3000\"\r\n\r\n    private val retrofit = Retrofit.Builder()\r\n        .baseUrl(BASE_URL)\r\n        .addConverterFactory(GsonConverterFactory.create())\r\n        .build()\r\n\r\n    val api: Api = retrofit.create(Api::class.java)\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/projecte12/RetroFit.kt b/app/src/main/java/com/example/projecte12/RetroFit.kt
--- a/app/src/main/java/com/example/projecte12/RetroFit.kt	(revision 78e43641e98053aba1698f4b7a56e0eee52a90ee)
+++ b/app/src/main/java/com/example/projecte12/RetroFit.kt	(date 1730201553916)
@@ -4,7 +4,7 @@
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetroFit {
-    private const val BASE_URL = "http://10.0.2.2:3000"
+    private const val BASE_URL = "http://10.0.2.2:3001"
 
     private val retrofit = Retrofit.Builder()
         .baseUrl(BASE_URL)
